app-commons/src/main/scala/org/bitcoins/commons/serializers/JsonWriters.scala

```diff
@@ -24,6 +24,7 @@ object JsonWriters {
 
     override def writes(hash: HashType): JsValue =
       hash match {
+        case SIGHASH_DEFAULT                => JsString("DEFAULT")
```

Comment:
Should start using this instead of `ALL` in a lot of places I think. Probably fine to do later though